cmd/snapctl/task.go

```diff
@@ -48,6 +48,9 @@ var (
 	unionParseFormat = timeParseFormat + " " + dateParseFormat
 )
 
+// Constants used to truncate task hit and miss counts
+// e.g. 1k, 1M, etc. As a side note, we probably don't need
+// anything after the M.
```

Comment:
K should be capitalized :). I'm not sure if @danielscottt really meant to have the side note as part of the comment.